### PR TITLE
RASS-9 prisoner banner on every page

### PR DESCRIPTION
**Pull request by carloveo-moj**

### CODE REVIEW

File: assets/images/prisoner-profile-image.png
Comment (carloveo-moj) [inline assets/images/prisoner-profile-image.png:1]: a default image taken from prisoner profile, in next PR will add getting the profile image from prison API

File: assets/scss/components/_prisoner-banner.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.dwf-header {
```
Comment (carloveo-moj) [inline assets/scss/components/_prisoner-banner.scss:1]: prototype styles

File: server/app.ts [hunk 2]
```diff
@@ -37,8 +38,8 @@ export default function createApp(services: Services): express.Application {
   app.use(authorisationMiddleware(['ROLE_REMAND_AND_SENTENCING', 'ROLE_CALCULATE_RELEASE_DATES']))
   app.use(setUpCsrf())
   app.use(setUpCurrentUser(services))
-
-  app.use(routes(services))
+  app.get('/person/:nomsId', setupPrisonerDetails(services.prisonerService))
```
Comment (carloveo-moj) [inline server/app.ts:41]: on every route which begins with /person/:nomsId retrieve the prisoner details

File: server/views/components/prisonerBanner/template.njk [hunk 3]
```diff
@@ -0,0 +1,23 @@
+
+<div class="govuk-grid-row">
```
Comment (carloveo-moj) [inline server/views/components/prisonerBanner/template.njk:2]: taken from prototype

File: server/views/partials/layout.njk [hunk 4]
```diff
@@ -28,6 +29,37 @@
 {% block bodyStart %}
 {% endblock %}
 
+ {% block main %}
```
Comment (carloveo-moj) [inline server/views/partials/layout.njk:32]: aligned layout with prototype

**Review by joelstobart-moj — APPROVED** [review]
LGTM

**Pull request merged**